cms/startup.py

```diff
@@ -17,6 +18,8 @@ def run():
     """
     django_utils_translation.patch()
 
+    django.setup()
```

Comment:
We may need to add it, it's my understanding that with Django 1.7 the way that the app cache is loaded has changed: https://docs.djangoproject.com/en/1.8/releases/1.7/#app-loading-refactor . What was happening here is that the models were being imported by the signal handlers before the app cache is ready. By calling setup() explicitly the app cache is loaded so the models that the signal handler needs are available. 

The reason we have to do this is because of django-simple-history which does some things to models as they are loaded which are not part of the publicly available django api's...

Check out this discussion to for more clarification:
https://github.com/treyhunner/django-simple-history/issues/104